### ISSUE TITLE
shotly.io

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#3161]
### Basic Information

**Test link:** `https://shotly.io/CFhHE`
**Category:** ads
**Reported from:** India
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:73.0) Gecko/20100101 Firefox/73.0
**Extension:** Nano Defender for Firefox 15.0.0.185

### Additional Message

Original URL: `about:newtab`

3rd party tracking before and after captcha completion

**Comment by JobcenterTycoon** [comment]
`||ofhypnoer.com^` is already blocked by easylist

**Comment by mapx-** [comment]
removed

**Comment by ghajini** [comment]
`https://shotly.io/aevk`
Fake recaptcha

**Comment by ghajini** [comment]
`https://shotly.io/Qnei9`
Popup ads

**Comment by jspenguin2017** [comment]
@ghajini Can't reproduce popup, opened a new issue for fake buttons: https://github.com/NanoMeow/QuickReports/issues/3909

**Comment by ghajini** [comment]
Yes that propeller popup code was there yesterday at that site,code was removed by site......

Here is new url which has this

`https://shotly.io/nfAS`

**Comment by jspenguin2017** [comment]
Still no popups for me.
